docs/docs/optics.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: docs
+title: Mu-Optics
+permalink: optics/
+---
+
+# Mu-Optics
+
+We created a new package `mu-optics` in the **release v0.3 of Mu-Haskell** to give an easier API to build servers and clients using lenses and prisms (probably the ultimate API 😉). This document aims to be a reference of how to use this new library and the common use cases for it.
+
+## Accessing fields with `#field`
+
+When you want to refer to a method of your type-level schema, whereas in the server or the client, you can use the getter lens (`^.`) in conjunction with the `OverloadedLabels` extension to access that field, as in the example:
```

Comment:
The phrase "getter lens" is not correct. You mean "the get operation from the lens" or something like that.